### PR TITLE
Fix figure numbers in ch1

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Updated figure numbers in first chapter.

### CODE REVIEW

File: docs/architecture/cloud-native/definition.md [hunk 1]
```diff
@@ -295,11 +293,10 @@ The [Twelve-Factor Application](https://12factor.net/), discussed earlier, calls
 
 Modern CI/CD systems help fulfill this principle. They provide separate deployment steps and help ensure consistent and quality code that's readily available to users. 
 
-Figure 1.6 shows the separation across the deployment process.
+Figure 1.8 shows the separation across the deployment process.
```
Comment (mvelosop) [inline docs/architecture/cloud-native/definition.md]: minor-minor issue
```suggestion
Figure 1-8 shows the separation across the deployment process.
```

**Review by mvelosop — COMMENTED** [review]
Hi @ardalis just a micro issue I fixed locally for generation.

File: docs/architecture/cloud-native/definition.md [hunk 2]
```diff
@@ -34,11 +34,10 @@ Here are some companies who have implemented these techniques. Think about the s
 
 As you can see, Netflix, Uber, and WeChat expose systems that consist of hundreds of independent microservices. This architectural style enables them to rapidly respond to market conditions. They can instantaneously update small areas of a live, complex application, and individually scale those areas as needed.
 
-The speed and agility of cloud native come about from a number of factors. Foremost is cloud infrastructure. Five additional foundational pillars shown in Figure 1-1 also provide the bedrock for cloud-native systems.
+The speed and agility of cloud native come about from a number of factors. Foremost is cloud infrastructure. Five additional foundational pillars shown in Figure 1-3 also provide the bedrock for cloud-native systems.
 
 ![Cloud-native foundational pillars](./media/cloud-native-foundational-pillars.png)
-
-**Figure 1-1**. Cloud-native foundational pillars
+**Figure 1-3**. Cloud-native foundational pillars
```
Comment (mairaw) [inline docs/architecture/cloud-native/definition.md]:
```suggestion

**Figure 1-3**. Cloud-native foundational pillars
```

File: docs/architecture/cloud-native/definition.md [hunk 3]
```diff
@@ -243,11 +242,10 @@ Azure Kubernetes Services is covered in detail Chapter 2, *Scaling Cloud-Native
 
 Cloud-native systems depend upon many different ancillary resources, such as data stores, message brokers, monitoring, and identity services. These services are known as [backing services](https://12factor.net/backing-services).
 
- Figure 1-5 shows many common backing services that cloud-native systems consume.
+ Figure 1-7 shows many common backing services that cloud-native systems consume.
 
 ![Common backing services](./media/common-backing-services.png)
-
-**Figure 1-5**. Common backing services
+**Figure 1-7**. Common backing services
```
Comment (mairaw) [inline docs/architecture/cloud-native/definition.md]:
```suggestion

**Figure 1-7**. Common backing services
```

File: docs/architecture/cloud-native/candidate-apps.md [hunk 4]
```diff
@@ -28,10 +28,10 @@ Then there are legacy systems. While we'd all like to build new applications, we
 
 ### Modernizing legacy apps
 
-The free Microsoft e-book [Modernize existing .NET applications with Azure cloud and Windows Containers](https://dotnet.microsoft.com/download/thank-you/modernizing-existing-net-apps-ebook) provides guidance for migrating on-premises workloads into cloud. Figure 1-8 shows that there isn't a single, one-size-fits-all strategy for modernizing legacy applications.
+The free Microsoft e-book [Modernize existing .NET applications with Azure cloud and Windows Containers](https://dotnet.microsoft.com/download/thank-you/modernizing-existing-net-apps-ebook) provides guidance for migrating on-premises workloads into cloud. Figure 1-10 shows that there isn't a single, one-size-fits-all strategy for modernizing legacy applications.
 
 ![Strategies for migrating legacy workloads](./media/strategies-for-migrating-legacy-workloads.png)
-**Figure 1-8**. Strategies for migrating legacy workloads
+**Figure 1-10**. Strategies for migrating legacy workloads
```
Comment (mairaw) [inline docs/architecture/cloud-native/candidate-apps.md]:
```suggestion

**Figure 1-10**. Strategies for migrating legacy workloads
```

File: docs/architecture/cloud-native/definition.md [hunk 5]
```diff
@@ -295,11 +293,10 @@ The [Twelve-Factor Application](https://12factor.net/), discussed earlier, calls
 
 Modern CI/CD systems help fulfill this principle. They provide separate deployment steps and help ensure consistent and quality code that's readily available to users. 
 
-Figure 1.6 shows the separation across the deployment process.
+Figure 1.8 shows the separation across the deployment process.
 
 ![Deployments Steps in CI/CD Pipeline](./media/build-release-run-pipeline.png)
-
-**Figure 1-6**. Deployment steps in a CI/CD Pipeline
+**Figure 1-8**. Deployment steps in a CI/CD Pipeline
```
Comment (mairaw) [inline docs/architecture/cloud-native/definition.md]:
```suggestion

**Figure 1-8**. Deployment steps in a CI/CD Pipeline
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good. Left a few comments in addition to the ones from @mvelosop.

**Comment by ardalis** [comment]
Hopefully all set now.

**Comment by mvelosop** [comment]
@ardalis, I'm a bit confused or perhaps I'm missing something. Did you mean to close this PR so as not to apply the changes?

**Comment by ardalis** [comment]
No, I must have hit the wrong button with my last comment.

**Review by mairaw — APPROVED** [review]
LGTM. Thanks @ardalis!

**Pull request merged**